wp-content/themes/citylimits/single-two-column.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Single Post Template: Two Column (Classic Layout)
+ * Template Name: Two Column (Classic Layout)
+ * Description: Shows the post and sidebar if specified.
+ *
+ * This template merges in portions of largo's partials/content-single-classic.php
+ * and partials/content-page.php because the desired layout puts things in places on the page
+ * that are not accomplishable without doing so:
+ * - putting the header above the span4/span8 split
+ */
+
+global $shown_ids;
+
+add_filter('body_class', function($classes) {
+	if ( is_page() ) {
+		$classes[] = 'hnews';
+		$classes[] = 'item';
+	}
+
+	$classes[] = 'classic row-fluid span12';
+	return $classes;
+});
+
+get_header();
+?>
+
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?> itemscope itemtype="https://schema.org/Article">
+	<?php if ( is_page() ) { do_action('largo_before_page_header'); } ?>
+	<header>
+
+		<h1 class="entry-title" itemprop="headline"><?php the_title(); ?></h1>
+		<?php
+			if ( $subtitle = get_post_meta( $post->ID, 'subtitle', true ) ) {
+				echo '<h2 class="subtitle">' . $subtitle . '</h2>';
```

Comment:
This can be an [`hgroup`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup), to simplify styling. The border-bottom goes on the hgroup, and the elements go within it.